recipe/meta.yaml

```diff
@@ -11,30 +11,34 @@ source:
 
 build:
   noarch: python
-  number: 0
+  number: 1
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - python >=3.7
     - pip
     - pytest-runner
+    - setuptools_scm
   run:
     - python >=3.7
     - xarray
     - pandas
     - numpy
+    - importlib-metadata
 
 test:
   imports:
     - cmip6_preprocessing
+    
+  commands: python -c "import cmip6_preprocessing; assert cmip6_preprocessing.__version__ == '{{ version }}', cmip6_preprocessing.__version__"
```

Comment:
Oh very nice! I have been struggling with this in other packages. This is super helpful